### ISSUE TITLE
Refused to connect

### ISSUE THREAD

**Issue by RizzleQ** [bokub/home-assistant-extension#16]
Been using this extension for months with no issue. Suddenly this started happening. Doesn't matter what URL I use (local hostname, IP address, internet domain HTTPS address), it's always refused. There's no failed login or any logs that I can see in Home Assistant that shows a problem. Any ideas where to start?

![2023-09-07 21_52_41](https://github.com/bokub/home-assistant-extension/assets/87519056/15e927b2-067d-498f-bd30-98c7f40b63d4)

**Comment by DuDe-75** [comment]
Same here...I'm not entirely sure but I think it is because some updates in Home Assistant. At least I see this error since about the time Home Assistant had a Supervisor update (version 2023.08.3). I'm using Home Assistant on a Raspberry Pi 4.

I could be wrong tho, but it would be nice to have a fix on this one.

**Comment by RizzleQ** [comment]
Hmm, I don’t recall precisely when this started happening. Was probably around a week and a half ago, which does coincide with when I upgraded to supervisor 2023.08.3.

I wonder if any of the changes made to that version could have an affect:

https://github.com/home-assistant/supervisor/releases/tag/2023.08.3

**Comment by bokub** [comment]
Hello !

I have Home assistant 2023.8.4 installed on my Pi and I don't have this problem...

**Comment by RizzleQ** [comment]
> 

That sounds like an older version of [Home Assistant Core](https://github.com/home-assistant/core/releases/tag/2023.8.4) from 2 weeks ago.

The latest version of Supervisor is [2023.09.0](https://github.com/home-assistant/supervisor/releases/tag/2023.09.0) which would be worth testing to see if it helps. It's not showing as an available update for me yet.

**Comment by bokub** [comment]
Oh I didn't look at the right version, my **supervisor** version is 2023.08.3, and I don't have the problem

**Comment by bokub** [comment]
@RizzleQ The extension is nothing more than a fancy `<iframe src="YOUR_URL" />`

Maybe look at the response headers of your URL and check if there are special headers such as [X-Frame-Options](https://developer.mozilla.org/fr/docs/Web/HTTP/Headers/X-Frame-Options) :shrug:

**Comment by RizzleQ** [comment]
@bokub I did some digging and this problem is caused by [this Breaking change in HA Core 2023.9](https://github.com/home-assistant/core/pull/97784).

If I disable <code>use_x_frame_options</code> by setting it to <code>false</code> in the <code>http</code> configuration, this problem is resolved :)

**Comment by RizzleQ** [comment]
Since this isn't a problem with this extension, I'll close this as resolved but @bokub you may want to let your users know about this change in HA so they can get this extension to work properly.

**Comment by bokub** [comment]
Thanks a lot @RizzleQ :+1:

**Comment by nomonkeynodeal** [comment]
Every time people in a thread come up with a solution for something and don't say what the solution is or how to access it, an angel dies.

**Comment by DuDe-75** [comment]
that is i right...and don't be alarmed when editing the config within Home Assitant using something like 'Studio Code Server', and you get a warning that "Propert use_x_frame_options is not allowed". Just check if it is added with the proper syntax (see the comment before this one), then restart Home assistant, and the extension will work ;)

**Comment by SmartLiving-Rocks** [comment]
Hello!
It must be something else - in my case. I am on a Mac 10.14.6  and just tested the extension with another user profile, and it worked directly. 

![Bildschirmfoto 2023-10-18 um 09 06 15](https://github.com/bokub/home-assistant-extension/assets/76279896/5c3e3204-db5c-461a-a8ec-8c2d27eb09f6)

On my main profile, I have noticed the issue after I have changed my password. So I assume that there is something password related cached somewhere. 
Of course, I have cleared the cache and deleted and reinstalled your browser extension!
Any ideas?

Thanks in advance
